src/main/java/co/rsk/federate/signing/hsm/message/PowHSMSignerMessage.java

```diff
@@ -18,33 +18,69 @@
 
 package co.rsk.federate.signing.hsm.message;
 
+import static co.rsk.federate.signing.HSMField.INPUT;
+import static co.rsk.federate.signing.HSMField.OUTPOINT_VALUE;
+import static co.rsk.federate.signing.HSMField.SIGHASH_COMPUTATION_MODE;
+import static co.rsk.federate.signing.HSMField.TX;
+import static co.rsk.federate.signing.HSMField.WITNESS_SCRIPT;
+
 import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Coin;
 import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.bitcoinj.core.TransactionWitness;
 import co.rsk.trie.Trie;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 import org.ethereum.core.TransactionReceipt;
 import org.spongycastle.util.encoders.Hex;
 
 public class PowHSMSignerMessage extends SignerMessage {
+
+    private static final int HSM_VERSION_5 = 5;
```

Comment:
I see this often, it might be a good idea to have an enum of different HSM versions so we are not using ints. Wdyt, maybe for a future task?

Reply:
That sounds good to me. I agree that this should be moved to the enum, where we define all the HSM versions that we support. Thanks for bringing this up. I will create a task.

Reply:
Task created.